tests/fixtures/addon/content-for-head/tests/acceptance/meta-tag-modules-test.js

```diff
@@ -0,0 +1,14 @@
+import Ember from 'ember';
+import { test } from 'qunit';
+import moduleForAcceptance from '../../tests/helpers/module-for-acceptance';
+
+moduleForAcceptance('Acceptance | meta tag modules');
+
+test('visiting /', function(assert) {
+  visit('/');
+
+  andThen(function() {
+    let result = Ember.$('#configvalue').html();
```

Comment:
Since this fails in both unit and acceptance tests I'm going to revert back to my unit test.